lib/supa/command.rb

```diff
@@ -16,8 +16,12 @@ def represent
     private
 
     def value
-      return @representer.send(@options[:modifier], apply_render_flags(raw_value)) if @options[:modifier]
-      apply_render_flags(raw_value)
+      return @value if defined?(@value)
+      @value = modifier ? @representer.send(modifier, apply_render_flags(raw_value)) : apply_render_flags(raw_value)
```

Comment:
To be honest, I don't quite like the part with `...= modifier ?...`, as it can return `true` for cases like `''` (empty string), I think we should fix it.